authz/build.gradle

```diff
@@ -3,15 +3,14 @@ plugins {
     id 'java-test-fixtures'
 }
 
-dependencies {
-    implementation(project(':common'))
+components.java.withVariantsFromConfiguration(configurations.testFixturesApiElements) {
+    skip()
```

Comment:
required to prevent publishing test-fixtures artefacts

Reply:
when I add this to conventions, the build starts failing as it is not able to recognise the  `testFixturesApiElements` configurations